.github/scripts/init_linux_port_vm_network.sh

```diff
@@ -3,39 +3,90 @@
 # Copyright (c) Microsoft. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
-# This script sets up virtual interface (veth1 and veth0 ) with DHCP configure that can be used for linux port.
-
-ip link add veth0 type veth peer name veth1
-ifconfig veth0 192.168.1.1 netmask 255.255.255.0 up
-ifconfig veth1 up
-ethtool --offload veth0 tx off
-ethtool --offload veth1 tx off
-
-sh -c "cat > /etc/dhcp/dhcpd.conf" <<EOT
-subnet 192.168.1.0 netmask 255.255.255.0 {
-    range 192.168.1.100 192.168.1.200;
-    option routers 192.168.1.1;
-    option domain-name-servers 1.1.1.1;
-    option broadcast-address 192.168.1.255;
-    default-lease-time 600;
-    max-lease-time 7200;
+# This script sets up virtual interface (rtosveth1 and rtosveth0) with DHCP configure that can be used for linux port.
+
+set -o errexit  # Exit if command failed.
+set -o nounset  # Exit if variable not set.
+set -o pipefail # Exit if pipe failed.
+
+getNextAvailableIpRange() {
+    for (( i=1; i<=255; i++ )); do
+        match=`ifconfig -a | grep -o -E "inet $1\.$2\.$i\.[0-9]+"`
+
+        if [[ "$match" == "" ]]; then
+            echo $i
+            break
+        fi
+    done
 }
+
+if [[ "$1" == "--help" || "$1" == "-h" ]]; then
```

Comment:
It looks like if you run this script after adding 8-10 lines with no arguments, script should fail with ": $1: unbound variable". Can you please confirm.

Reply:
Fixed.